crates/router/tests/connectors/selenium.rs

```diff
@@ -47,13 +46,28 @@ pub enum Assert<'a> {
     Contains(Selector, &'a str),
     ContainsAny(Selector, Vec<&'a str>),
     IsPresent(&'a str),
+    IsElePresent(By),
     IsPresentNow(&'a str),
 }
 
 pub static CHEKOUT_BASE_URL: &str = "https://hs-payments-test.netlify.app";
-pub static CHEKOUT_DOMAIN: &str = "hs-payments-test.netlify.app";
 #[async_trait]
 pub trait SeleniumTest {
+    fn get_saved_testcases(&self) -> serde_json::Value {
+        let env_value = env::var("CONNECTOR_TESTS_FILE_PATH").ok();
+        if env_value.is_none() {
+            return serde_json::json!("");
+        }
+        let path =
+            env::var("CONNECTOR_TESTS_FILE_PATH").expect("connector tests file path not set");
```

Comment:
What exactly is the intended behavior?

Reply:
Connector testcases are maintained outside the repo, if this path is not provided manually the UI tool will try to download the testcases each time we run the tests, we are avoiding the extra call here